src/actions/make/_make.lua

```diff
@@ -144,8 +144,17 @@
 		_p('')
 
 		if kind == "workspace" then
-			_p('.NOTPARALLEL:')
-			_p('')
+			local haspch = false
+			for _, prj in ipairs(target.projects) do
+				if prj.pchheader then
```

Comment:
unfortunately the scope of pchheader == config... 
```lua
	api.register {
		name = "pchheader",
		scope = "config",
		kind = "string",
		tokens = true,
	}
```

meaning that looking at this value at the project level is insufficient. 

```lua
project 'a'
    filter { 'configuration:debug' }
        pchheader 'b.h'
```

haspch would be false in this case.